SConstruct

```diff
@@ -282,11 +282,14 @@ opts.Add(BoolVariable("builtin_zstd", "Use the built-in Zstd library", True))
 opts.Add("CXX", "C++ compiler binary")
 opts.Add("CC", "C compiler binary")
 opts.Add("LINK", "Linker binary")
+opts.Add("asflags", "Custom flags for the assembler")
+opts.Add("arflags", "Custom flags for the archive tool")
 opts.Add("cppdefines", "Custom defines for the pre-processor")
 opts.Add("ccflags", "Custom flags for both the C and C++ compilers")
 opts.Add("cxxflags", "Custom flags for the C++ compiler")
 opts.Add("cflags", "Custom flags for the C compiler")
 opts.Add("linkflags", "Custom flags for the linker")
+opts.Add("rcflags", "Custom flags for Windows resource compilers")
```

Comment:
I'd put the 3 ones after `linkflags`, as I think `cppdefines`, `c*flags`, and `linkflags` are the most likely to be interesting to users, in this order.

Reply:
Done